com.stansassets.google-doc-connector-pro/Runtime/Localization/LocalizationClient.cs

```diff
@@ -348,6 +373,14 @@ public string GetLocalizedString(ILocalizationToken localizationToken, string la
 
             return finalText;
         }
+        
+        /// <summary>
+        /// Notify that the language settings of the spreadsheet have been changed
+        /// </summary>
+        void NotifyLanguageChanged()
```

Comment:
```suggestion
        void OnDataStoredOnDiskHandler()
```
The callback method name is out of our convention. Please, consider renaming.